webpack.config.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+// generated on 2017-09-01 using generator-openmrs-owa 0.6.0
+
+const webpack = require('webpack');
+const path = require('path');
+const fs = require('fs');
+const env = require('yargs').argv.mode;
+const target = require('yargs').argv.target;
+const targetPort = require('yargs').argv.targetPort;
+
+const UglifyPlugin = webpack.optimize.UglifyJsPlugin;
+const CommonsChunkPlugin = webpack.optimize.CommonsChunkPlugin;
+const DedupePlugin = webpack.optimize.DedupePlugin;
+
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const BrowserSyncPlugin = require('browser-sync-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const WebpackOnBuildPlugin = require('on-build-webpack');
+
+const nodeModulesDir = path.resolve(__dirname, '../node_modules');
+
+require.extensions['.webapp'] = function(module, filename) {
+  module.exports = fs.readFileSync(filename, 'utf8');
+};
+const manifest = require('./app/manifest.webapp');
+
```

Comment:
It looks like you are grouping statements by their functionality but some line spaces in this file are unnecessary. Look into it.

Reply:
I guess the OpenMRS OWA generator should be fixed, haha. I think sometimes line spaces are okay because some developers don't like code that is too squashed and there is no rule against it. Visit [this](https://eslint.org/docs/rules/key-spacing) for some context.